packages/create-plugin/scripts/generate-panel.js

```diff
@@ -11,8 +11,8 @@ async function generatePanel() {
   const plop = await nodePlop(plopFile);
   const generator = plop.getGenerator('create-plugin');
   await generator.runActions({
-    pluginName: 'my-plugin',
-    orgName: 'my-org',
+    pluginName: 'myplugin',
```

Comment:
Rather than change these shouldn't we do something with `normalize_id` so whatever the input is the output is schema compliant?

Reply:
good idea, will update my PR

Reply:
These inputs are equivalent to user input.